types/pigpio-dht/index.d.ts

```diff
@@ -0,0 +1,46 @@
+// Type definitions for pigpio-dht 1.1
+// Project: https://github.com/depuits/pigpio-dht
+// Definitions by: Erik Mavrinac <https://github.com/erikma>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="node" />
+
+export = dht;
+
+/**
+ * Returns a new DHT object for accessing a DHT11 or DHT22/AM2302 via GPIO.
+ * @param gpio      an unsigned integer specifying the GPIO number
+ * @param type      A DhtType enumeration value for the sensor type
+ */
+declare function dht(gpio: number, type: dht.DhtType | number): dht.Dht;
+
+declare namespace dht {
+    /** The various sensor types supported by pigpio-dht. */
+    enum DhtType {
+        DHT11 = 11,
+        DHT22 = 22,
+        AM2302 = 22,
```

Comment:
If these values are identical, I would suggest setting aliases to the primary enum member, like `AM2302 = DHT22`, instead of repeating the value each time.